releasenotes/notes/dd-trace-methods-wrapping-async-functions-efa8599ac82c45f2.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: This fix resolves an issue where asynchronous functions instrumented by `DD_TRACE_METHODS` ended their spans prematurely (right after coroutine creation).
+    The updated logic properly awaits the async call, ensuring the span remains open until the coroutine finishes.
```

Comment:
Spans from DD_TRACE_METHODS don't usually trigger metrics, but can in some cases. It's worth adding some notes that this will impact metrics based on this span duration since this will affect latency in the cases where we get metrics from these spans.